libs/elements/src/lib/select/vl-select.lib.js

```diff
@@ -4212,9 +4213,15 @@
             placeholderValue = vl.i18n.t('select.placeholder_value'),
             disabled = false; // Select has search - default behavior
 
+        // @deprecated: This is not how boolean attributes should work
```

Comment:
best op deze manier deprecaten
/** @deprecated: This is not how boolean attributes should work */